lib/SPIRV/libSPIRV/SPIRVValue.cpp

```diff
@@ -124,6 +89,40 @@ void SPIRVValue::setFPFastMathMode(SPIRVWord M) {
                      << "\n")
 }
 
+template <spv::Decoration NoIntegerWrapDecoration>
+void SPIRVValue::setNoIntegerDecorationWrap(bool HasNoIntegerWrap) {
```

Comment:
What is the purpose of HasNoIntegerWrap parameter? Calls to this functions are added only with true parameter and under if checking that the LLVM value has "NoIntegerWrap".

Reply:
I also find this strange, but prefer not to change it within this patch set (even if it already brings some code outside of the declared purpose).

Reply:
:(